hashicorp/boundary-ui
# refactor: 💡 Remove route-action usage for managed-groups

✅ Closes: https://hashicorp.atlassian.net/browse/ICU-12619

## Description


Remove route-action usage for managed-groups

## How to Test


1. Run a boundary dev instance.
2. Use existing OIDC and LDAP auth-methods.
3. Test CRUDL operations for managed-groups (create, edit, save, cancel, list, delete)
4. Make sure there are no errors or new warnings in dev tools console.


:technologist: [Admin preview](PATH_TO_FEATURE)

:desktop_computer: [Desktop preview](PATH_TO_FEATURE)

## Checklist:
